### DISCUSSION TITLE
About my Student developer package with Copilot access

### DISCUSSION THREAD

**Discussion by KavindaRajapaksha** [community/community#73035] [GitHub Education]
### Select Topic Area

Question

### Body

I have a Student developer package approved account then can I have get GitHub copilot feature free?, If I can get copilot access free how can I activate it now I have free one month trial version of GitHub copilot

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @KavindaRajapaksha , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
